platform/ios/docs/guides/Adding Points to a Map.md

```diff
@@ -1,82 +1,58 @@
+## **Annotations API**
 
-## MGLPointAnnotation
+Our annotations API includes the `MGLAnnotationImage`, and `MGLAnnotationView` classes. These are most similar to MapKit’s annotation class and provide a familiar interface for working with markers and callouts.
 
-It’s straightforward to add an annotation to a map. You can use `MGLPointAnnotation` as is, or you can subclass it to add annotations with richer data.
+| ![MGLAnnotationImage](img/adding-points-to-a-map/annotation-image.png "MGLAnnotationImage")               | ![MGLAnnotationView](img/adding-points-to-a-map/annotation-view.png "MGLAnnotationView")              |
+|----------------------|---------------------|
+| `MGLAnnotationImage` | `MGLAnnotationView` |
 
-```swift
-let annotation = MGLPointAnnotation()
-annotation.coordinate = CLLocationCoordinate2D(latitude: 45.5076, longitude: -122.6736)
-annotation.title = "Bobby's Coffee"
-annotation.subtitle = "Coffeeshop"
-mapView.addAnnotation(annotation)
-```
+**MGLAnnotationImage** is an annotation class that is easily customizable with any `UIImage`.
+It is highly performant, as the images are rendered directly in OpenGL. However, if you need to animate your annotations or control z-layer ordering, consider working with **MGLAnnotationView** which supports basic CALayer animations and can be stacked by using `zPosition` property on `CALayer`.
 
-See the `MGLMapViewDelegate` method `-mapView:annotationCanShowCallout:` and similar methods for allowing interaction with a callout ([example](https://www.mapbox.com/ios-sdk/examples/callout-delegate/)).
+**MGLAnnotationView** is an annotation class that is easily customizable with a `UIView`. Use this class if you need your markers to be dynamic or animatable. MGLAnnotationView has a significant advantage over MGLAnnotationImage when you need every annotation to be unique. For example, annotation views are ideal for showing user locations on a map using high-resolution profile pictures. However, the map can get slow down when many annotation views are visible at the same time, so if you need to add a very large amount of markers, consider using our runtime styling APIs instead.
 
-## Displaying annotations
+Both MGLAnnotationImage and MGLAnnotationView can become interactive with the addition of a single line of code. When the user taps an annotation, the annotation’s name appears in a basic callout. An annotation view can additionally respond to drag-and-drop gestures.
 
-There are two basic ways to display the annotations you’ve added to a map, each with their own tradeoffs.
+## **Runtime Styling API**
 
-### Annotation Images (`MGLAnnotationImage`)
+For absolute full control of how points are displayed on a map, consider using our [runtime styling](#) APIs. It is the most performant approach for adding hundreds of markers because markers are rendered in GL directly. The runtime styling API provides support for labels rendered together with icons, finer control of z-ordering, and clustering.
```

Comment:
> It is the most performant approach for adding hundreds of markers because markers are rendered in GL directly.

This sentence is a slightly differently worded version of the one in `MGLAnnotationImage` that uses the same explanation, without clarifying the difference/similarity. It might be good to relate Runtime Styling directly to `MGLAnnotationImage`, saying that both are performant because they’re “rendered directly using OpenGL”.